data-catalog-policies-app/src/main/resources/application-test.properties

```diff
@@ -15,4 +15,6 @@ spring.flyway.schemas=policy_schema
 spring.flyway.baseline-on-migrate=true
 
 # Endpoints
-datacatalog.codelist.purpose.url=http://35.201.118.102/backend/codelist/PURPOSE
+datacatalog.codelist.url=http://35.201.118.102/backend/codelist
```

Comment:
We can use a local config file which can hold IP's. The local config file can be part of a private repo. We can do it in a separate issue.